packages/vite/src/node/plugins/css.ts

```diff
@@ -1478,7 +1488,7 @@ async function rebaseUrls(
   }
 
   if (hasUrls) {
-    rebased = await rewriteCssUrls(rebased || content, rebaseFn)
+    rebased = await rewriteCssUrls(rebased || content, rebaseFn, file)
```

Comment:
Maybe we could pass `variablePrefix`(`$`, `@`) from `rebaseUrls` is called instead of file name? It will remove many conditions.